### PR TITLE
Fixed error message, remove SymbolPaths.txt

### PR DESCRIPTION
**Pull request by antonrohr**
http://b/226103990

### CODE REVIEW

File: src/OrbitGl/App.cpp [hunk 1]
```diff
@@ -1850,10 +1850,9 @@ orbit_base::Future<ErrorMessageOr<std::filesystem::path>> OrbitApp::RetrieveModu
                                                     debuglink.crc32_checksum, GetAllSymbolPaths());
         if (local_debuginfo_path.has_error()) {
           return ErrorMessage{absl::StrFormat(
-              "Module \"%s\" doesn't include debug info, and a separate "
-              "debuginfo file wasn't found on this machine, when searching "
-              "the paths from your SymbolsPath.txt. Please make sure the "
-              "debuginfo file can be found in one of the listed directories. According to "
+              "Module \"%s\" doesn't include debug info, and a separate debuginfo file wasn't "
+              "found on this machine, when searching the folders from the Symbol Locations. Please "
+              "make sure the debuginfo file can be found in one of the added folders. According to "
```
Comment (dpallotti) [inline src/OrbitGl/App.cpp:1855]: "make sure that"?

**Review by dpallotti — APPROVED** [review]

File: src/OrbitGl/App.cpp (hunk 1 above)
Comment (antonrohr) [reply src/OrbitGl/App.cpp]: sgtm!

**Comment by antonrohr** [comment]
> LGTM. This is just changing a string. Should this go into the release branch?

We could put it in, but we have the symbol locations dialog since a couple of months now and I dont think its important enough to trigger a new build

**Comment by beckerhe** [comment]
> > LGTM. This is just changing a string. Should this go into the release branch?
> 
> We could put it in, but we have the symbol locations dialog since a couple of months now and I dont think its important enough to trigger a new build

Oh yeah. You're right. The dialog has already been there for quite a while. I somehow thought this was related to your recent improvements.

**Pull request merged**